modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -230,21 +244,32 @@ class ConnectionAvgStats {
 
                 this._avgRTT.appendReport(batchReport, props);
 
-                // Report end to end RTT only for JVB
-                if (!this.isP2P) {
-                    const avgRemoteRTT = this._calculateAvgRemoteRTT();
-                    const avgLocalRTT = this._avgRTT.calculate();
+                if (this.isP2P) {
+                    // Report RTT diff only for P2P.
+                    const jvbEnd2EndRTT = this._conference.avgRtpStatsReporter
```

Comment:
do we have to access it through the conference object ?
Maybe we could make 'AvgRTPStatsReporter' a parent (pass in the constructor) then this will give access to _conference and other stuff we may need in future